rgw/v2/tests/s3_swift/reusables/bucket_policy_ops.py

```diff
@@ -155,8 +157,26 @@ def DeleteObject(**kw):
     log.info(f"objects_delete_response: {objects_delete_response}")
     objects_delete_status = classify_response(objects_delete_response)
     if effect == "Allow":
+        if objects_delete_response:
+            objects_deleted_count = len(objects_delete_response["Deleted"])
+            log.error(f"expected number of objects deleted: {objects_count}")
+            log.error(f"actual number of objects deleted: {objects_deleted_count}")
+            if objects_deleted_count != objects_count:
+                objects_delete_status = False
+            else:
+                objects_delete_status = True
         return object_delete_status and objects_delete_status
     else:
+        if objects_delete_response:
+            deletion_error_count = len(objects_delete_response["Errors"])
+            log.error(f"expected number of object deletion errors: {objects_count}")
+            log.error(
+                f"actual number of object deletion errors: {deletion_error_count}"
+            )
+            if deletion_error_count != objects_count:
```

Comment:
can we use ternary operator here as well

Reply:
done @anrao19 
pls review again